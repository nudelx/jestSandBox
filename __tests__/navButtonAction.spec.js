import  NavButtonActionsCreator from  '../src/redux/action/navButtonAction'
import enzymeSerializer from 'enzyme-to-json/serializer';


describe('it should create navigation action ', () => {

  it('NavButtonActionsCreator', () => {
    expect.addSnapshotSerializer(enzymeSerializer)
    const action = NavButtonActionsCreator.createNavButtonClick('page_1')
    expect(action).toMatchSnapshot()
  })
})
