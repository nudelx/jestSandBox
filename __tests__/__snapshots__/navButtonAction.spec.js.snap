// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should create navigation action  NavButtonActionsCreator 1`] = `
Object {
  "page": "page_1",
  "type": "NAV_BUTTON::CLICK",
}
`;
